modules/hostap/Kconfig

```diff
@@ -170,4 +170,5 @@ config LOG_BUFFER_SIZE
 
 config WPA_SUPP_NO_DEBUG
 	bool "Disable printing of debug messages, saves code size significantly"
+	default y
```

Comment:
This should be a separate commit.

Reply:
And I don't think default should be no debugs, this hinders customer support.

Reply:
In that case, shall we enable this specifically in all matter related samples for now?

Reply:
Yes, each sample can decide individually.

Reply:
Done

Reply:
Right. This was inadvertantly included. Removed now.